packages/visual-editor/src/hooks/useEditorSubscriber.ts

```diff
@@ -183,6 +183,8 @@ export function useEditorSubscriber() {
           break;
         }
         case INCOMING_EVENTS.DesignComponentsRegistered:
+          // Event was deprecated and support will be discontinued with version 5
+          break;
```

Comment:
The web app will send both for the time being. So I figured it would work nicely by ignoring the old event type for newer versions of the SDK while older SDK versions only "understand" the old event type.